Add missing CHANGELOG entry for problem details (#1058)
inrupt/solid-client-notifications-js: CHANGELOG.md

The following changes have been implemented but not released yet:

> ### Deprecations
> 
> - `FetchError` is no longer thrown, and replaced with `ClientHttpError`.

Author: ```suggestion
### Breaking change

- `FetchError` is no longer thrown, and replaced with `ClientHttpError`. Please prefer using type guards from `@inrupt/solid-client-errors`, e.g. `hasErrorResponse` and `hasProblemDetails`, rather than `instanceof`.
```

> - `FetchError` is no longer thrown, and replaced with `ClientHttpError`. Please prefer using type guards from `@inrupt/solid-client-errors`, e.g. `hasErrorResponse` and `hasProblemDetails`, rather than `instanceof`.

Reviewer: ```suggestion
- `FetchError` is no longer thrown, and replaced with `ClientHttpError`. Please prefer using type guards from
  `@inrupt/solid-client-errors`, e.g. `hasErrorResponse` and `hasProblemDetails`, rather than `instanceof`.
```

### New Features

- Node 22 is now supported
- `ClientHttpError` is thrown on HTTP error. It implements `WithErrorResponse` and `WithProblemDetails`,

>   meaning it has respectively a `.errorResponse` and a `.problemDetails` getters. The former provides

Reviewer: ```suggestion
  meaning it has respectively `.errorResponse` and `.problemDetails` getters. The former provides
```
